clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -646,14 +659,49 @@ check_duplicate_syscall_with_same_pc()
     return true;
 }
 
+bool
+check_rseq_side_exit_discontinuity()
+{
+    // Negative test: Missing instructions in a basic block due to rseq side exit.
```

Comment:
Add a TODO to use the encoder in other tests.

Reply:
Done.